pagespeed-insights/src/appsscript.json

```
@@ -6,8 +6,8 @@
     "logoUrl": "https://www.gstatic.com/images/icons/material/product/2x/pagespeed_64dp.png",
     "company": "Google Data Studio Developer Relations",
```

Grant added this one, correct? If so...

```suggestion
    "company": "Grant Kemp",
```